examples/python/epem_to_chicpipi.py

```diff
@@ -1,37 +0,0 @@
-r"""Sample script for the testing purposes.
```

Comment:
why delete the python scripts?

Reply:
Oh thought those were the ones that were outdated? For sure they are not under CI

Reply:
Okay seems they do work actually. So, shall I put them back? And, if so, should a Travis job be made that tests them?

Reply:
I guess they are not in CI because they are slow? Normally this would still run though or serve as some kind of example. The xml files are recipes for tensorwaves, so they def do not belong here.

Reply:
Hmm i think the tricky part is the speed. They are just too slow for CI. Not sure if they have any value as an example for a user...

Reply:
Yeah, the notebooks already serve the purpose of documentation

Reply:
If we do remove the Python scripts, I should also move `examples/jupyter` --> `examples`

Reply:
Actually it's only `epem_to_d0d0barpipi.py` that takes ages

Reply:
all right, lets remove them then